articles/TOC.yml

```diff
@@ -50,24 +58,10 @@
     href: libraries/toc.yml
   - name: Quantum simulators and resource estimators
     href: machines/toc.yml
-  - name: Quantum computing concepts
-    href: concepts/toc.yml
-- name: How-to guides
-  items:
-  - name: Create a Q# project
-    href: how-to-guides/create-qsharp-project.md
-- name: Q# Learning Resources
-  items:
-  - name: Learn with the Quantum Katas
-    href: intro-to-katas.md
-  - name: Q# Samples
-    href: https://docs.microsoft.com/en-us/samples/browse/?languages=qsharp
+- name: Theoretical concepts
+  href: concepts/toc.yml
 - name: Resources
```

Comment:
Why do we need this section? Everything is resources, so this section only acts as a click-wall that provides zero information about what's inside.  For the release notes, we can mimic C# documentation and create a section called "What's new in the QDK" (https://docs.microsoft.com/en-us/dotnet/csharp/), while "License" and "Contributing to the QDK" can get their own section at the end of the TOC.

Reply:
Totally agree. I think the section came from the earlier days, and included the "Glossary" of quantum computing terms and something else, which both got sent to more fitting sections. Now it seems very not fitting with the title "Resources"